RecoLocalCalo/HGCalRecAlgos/interface/HGCalImagingAlgo.h

```diff
@@ -164,12 +164,11 @@ static const unsigned int maxlayer = 52;
 
 
 private:
-
 // last layer per subdetector
 static const unsigned int lastLayerEE = 28;
 static const unsigned int lastLayerFH = 40;
 // maximum number of wafers per Layer: 666 (V7), 794 (V8)
-static const unsigned int maxNumberOfWafersPerLayer = 794;
+static const unsigned int maxNumberOfWafersPerLayer = 796;
```

Comment:
Please update the text in the comments above: it still says 794. If instead this 796 corresponds to 794 precisely, please add a few words why.